portal/auth.py

```diff
@@ -0,0 +1,64 @@
+import functools
+
+from flask import (
+    Blueprint, flash, g, redirect, render_template, request, session, url_for
+)
+from werkzeug.security import check_password_hash, generate_password_hash
+
+from . import db
+
+bp = Blueprint('auth', __name__, url_prefix='/auth')
+
+@bp.route('/login', methods=('GET', 'POST'))
+def login():
+    email = request.args.get('email')
+    password = request.args.get('password')
```

Comment:
Form submission, especially if it has user credentials, should be a POST request, not a GET. You don't want to send passwords in plain text, and with GET it's appended to the URL as query params and can't be encrypted even over HTTPS. Change the form method to post and update this logic to match.